pkgs/development/python-modules/mocket/default.nix

```diff
@@ -72,6 +74,9 @@ buildPythonPackage rec {
     "test_real_request_session"
     "test_asyncio_record_replay"
     "test_gethostbyname"
+    "test_http_session"
+    "test_https_session"
+    "test_httprettish_session"
```

Comment:
Set `__darwinAllowLocalNetworking = true;` instead.

Reply:
Thanks, I didn't know about that! However, I'm not sure if it's even a Darwin-specific issue anymore -see below